packages/eas-cli/src/commands/build/configure.ts

```diff
@@ -77,6 +79,11 @@ export default class BuildConfigure extends EasCommand {
           await syncIosUpdatesConfigurationAsync(graphqlClient, projectDir, exp, projectId);
         }
       }
+
+      // configure for EAS Update
+      if (exp.updates?.url === getEASUpdateURL(projectId)) {
```

Comment:
move this into a method called something like `isProjectUsingEASUpdate` to ensure we use the same definition of what it means to be using EAS Update everywhere.